src/index.html

```diff
@@ -2,13 +2,108 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      rel="stylesheet"
+      href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="logo">
+            <a
+              href="#home"
+              class="logo"
+            >
```

Comment:
optional:
If the HTML-element has long attribute values or number of attributes is more than 2 - start each one, including the first, on the new line with 2-space indentation related to tag. (so, 2 or less, if their length is less than the maximum allowable, you do not need to move to a new line)

Reply:
If you don’t mind, I’ll leave it as it is, because I use automatic formatting from Vscode, which does not allow to limit the number of attributes on a new line, but speeds up the writing of code. Maybe there are some additional settings or plugins that I don’t know about. I would appreciate any advice on how to better tune it.😊